test/src/unit-capi-consolidation.cc

```diff
@@ -7139,10 +7139,13 @@ TEST_CASE_METHOD(
   if constexpr (tiledb::platform::is_os_windows) {
     return;
   }
-
   if (!vfs_test_setup_.is_local()) {
     return;
   }
+  char* manylinux_var = getenv("TILEDB_MANYLINUX");
```

Comment:
https://app.shortcut.com/tiledb-inc/story/47270/remove-manylinux-exclusion-from-vacuum-tests-in-tiledb-pr4889